### ISSUE TITLE
Going rogue

### ISSUE THREAD

**Issue by sacdallago** [PokemonGoers/Catch-em-all#134]
Dear @PokemonGoers/catch-em-all ,

I apologize for the silenze these past two days. I'm working heavily to get the site up and running, and unfortunately we didn't have any luck with AWS (with whom I'm still passive-agressively fighting over fax).

BUT.
BUT.

I have found a fallback solutions for two months or so. It's not 100%, I'm still fixing issues here and there with the API docker and bindings (eg. Mongo).
# 🎉  https://predictemall.online 🎉

Now it's time for us to go rouge! Please make sure that:
- [ ] All references to TUM, RostLab,  the Technical University of Munich/Technische Universitaet Muenchen etc are **removed** from the website. You can change phrases like "in the context of a course held at the.." to "in the context of a school project.." (in US english, school = college = everything). This applies to the about page.
- [ ] All references to PokemonGo and Niantic are **removed**. This should ensure that they cannot appeal to us stealing property, which we don't, since our predictions are completely third-party. But, just to make sure. This applies to the about page.
- [ ] Remove the _Imprint_ section from _Imprint & Disclaimer_ and, rename the page _Disclaimer_ only :)

Thank you :)

**Comment by MajorBreakfast** [comment]
What about mentions of Iteratec?

**Comment by gyachdav** [comment]
Iteratec will get back to us about that.

**Comment by johartl** [comment]
What should we do with the following section from the About page? Remove it entirely?

> On July 6th of this year, Niantic released the new location-based augmented reality title PokemonGo. The game took the world by storm and is now installed on 5% of smartphones in the US. In the game, players are using their smartphones to locate Pokemon - cute and cuddly virtual creatures that after being captured and nurtured turn into fearless fighters in the name of your ambition to level up.
> 
> In the real world Pokemon appear at random places and only for a short period of time. While firing up your PokemonGo app and starting to randomly look for a Pokemon that may pop up nearby may be a fun thing to do, we thought it could be nice if we do not leave things to random chance. Instead, we are intelligently guessing when and where our Pokemon will pop

**Comment by WoH** [comment]
WIP: https://github.com/PokemonGoers/Catch-em-all/tree/going_rogue

**Comment by WoH** [comment]
@johartl https://github.com/PokemonGoers/Catch-em-all/commit/b4595729dd0a59e95b2426df5b671e88b8dd9c39

**Comment by WoH** [comment]
- [Disclaimer](https://github.com/PokemonGoers/Catch-em-all/blob/going_rogue/ionic2/src/pages/imprint/imprint.page.html)
- [About](https://github.com/PokemonGoers/Catch-em-all/blob/going_rogue/ionic2/src/pages/about/about.page.html)

Anything we should do about [these](https://github.com/PokemonGoers/Catch-em-all/blob/going_rogue/ionic2/src/services/project-groups.service.ts#L15) links?

**Comment by WoH** [comment]
@sacdallago @gyachdav @goldbergtatyana @juanmirocks 
- [Disclaimer](https://github.com/PokemonGoers/Catch-em-all/blob/going_rogue/ionic2/src/pages/imprint/imprint.page.html)
- [About](https://github.com/PokemonGoers/Catch-em-all/blob/going_rogue/ionic2/src/pages/about/about.page.html)

Is this fine?

Anything we should do about [these](https://github.com/PokemonGoers/Catch-em-all/blob/going_rogue/ionic2/src/services/project-groups.service.ts#L15) links?

**Comment by sacdallago** [comment]
Looks fine to me.

Links can stay, @gyachdav ?

**Comment by gyachdav** [comment]
Still waiting for Iterate. Stand by for updates.

**Comment by gyachdav** [comment]
Here is the response from Iteratec:
- [ ]  it is important that the Liability Notice with reference to the person responsible is included (as present at the moment). Stand by to decision who the responsible person may be.
- [ ] Regarding  the mentioning of iteratec and the usage of the icon, the following is okay:
  a)       In the burger menu at the bottom  the logo can be placed together with the text “Hosting sponsored by iteratec”.
  b)      The part in the section “About” is ok.
- [ ] The logo cannot appear without the mentioning about the sponsoring. Also, the placement of the logo at the geo locations of the iteratec offices is not permissible.

**Comment by gyachdav** [comment]
@johartl @WoH @MajorBreakfast please bring this to a close. We want to launch the app.

**Comment by WoH** [comment]
@gyachdav Does this make sense while the PokeData API is down and the socket.io integration by is still messed up.

**Comment by WoH** [comment]
Also, who is this person:

>  it is important that the Liability Notice with reference to the person responsible is included (as present at the moment). Stand by to decision who the responsible person may be.

**Comment by sacdallago** [comment]
@WoH yeah. Sorry for that, I'm trying to push the two teams (predictions and data). The API will be up soon again, hopefully!

That person shall be @Georrgi :D He won't mind! ahahahahahahahahahaahahah JK, figuring out with @gyachdav and @goldbergtatyana :D

**Comment by WoH** [comment]
Lmao.

In all seriousness, please make sure that we can reach the data api and ideally, it would be nice if it would also have actual data to get there. I can start mocking everything for final testing purposes, but I don't like that idea too much.

I can finalize everything on the legal side within one hour.

**Comment by sacdallago** [comment]
@WoH data. Data is an issue. I'll check back with A if they have any clue what the status on bypassing the new firewalls on niantic's side are

**Comment by WoH** [comment]
@sacdallago If you look at the sources listed by the @PokemonGoers/pokedata Team, there is plenty of data to get there ;)
